### DISCUSSION TITLE
Linux Mint Future DEV

### DISCUSSION THREAD

**Discussion by sayeedbaksh** [linuxmint/discussions#228] [Ideas]
Why don't the Linux dev make a version like windows?

Microsoft will eventually switch over, use existing linux technology, because they save 10s of millions in labour, then add a skin over with snooping tech to datamine. 

If linux dev charged me $30 a year for linux, I would benefit, they would benefit and together we could fock microsoft. 

Not everyone is good at coding, not everyone have time. Why not make the community better?

Microsoft is doing a version of human trafficing except they do this with 1 and 0. it's enslavement. @clefebvre

**Comment by Secret-chest** [comment]
The GNU/Linux system is free software, there are paid distributions, because you are free to do that, the same way Mint is free to make it available without fee.

Linux Mint is pretty stable already, and the projects it depends on won't dissapear because they receive funding from large companies.

**Reply by sayeedbaksh** [reply]
I think you miss the point.

**Reply by Secret-chest** [reply]
GNU, Linux, Debian, APT, git, Python, GTK are all developed by the community and have no central company.
